src/mappings/poolController.ts

```diff
@@ -404,3 +404,31 @@ export function handleTransfer(event: Transfer): void {
 
   pool.save();
 }
+
+/************************************
+ ************* FXPOOL ***************
+ ************************************/
+
+export function handleAssimilatorIncluded(event: ethereum.Event): void {
```

Comment:
```suggestion
export function handleAssimilatorIncluded(event: AssimilatorIncluded): void {
```

Running `yarn codegen` generates the typings for all events, then you can access the assimilator with `event.params.assimilator`.

Reply:
Updated on [2152b9](https://github.com/balancer-labs/balancer-subgraph-v2/pull/291/commits/2152b96f9618837ec56dae15aba2f2ee44efd52f) @mendesfabio  ^^